lib/shared/utils.js

```diff
@@ -124,6 +124,18 @@ function runScript(command, args) {
     });
 }
 
+function getDataPlaceholder() {
```

Comment:
Why we need a function instead of const values?

Reply:
Fixed